pyfaidx/__init__.py

```diff
@@ -293,7 +293,7 @@ def __init__(self, filename, default_seq=None, key_function=lambda x: x.split()[
         self.filename = filename
 
         filenameLower = filename.lower()
-        if filenameLower.endswith('.bgz'):
+        if filenameLower.endswith('.bgz') or filenameLower.endswith('.gz'):
```

Comment:
I'm in favor of creating new locals if I think the definition of the local could change in time, so leveraging code reuse to reduce errors. In this case I'd be in favor of calling the function because:

- `filenameLower` is about as long as `filename.lower()`
- I'm not too worried about optimizing the speed of initializing `Faidx`
- I'm generally biased against local variables that are derived from function names. It makes me think of things like `my_list = list()` 

I'd stress that I'm less worried in this case about creating the local variable and would be okay with just changing `filenameLower` to `filename_lower`. Your choice.